Release of v.1.1.0

tibold opened the pull request:
Fixes:
* Asked vcredist not to reboot after installation
* Fixed locating the deployed qt plugins
* Fixed up CI builds
* Added support for GPG signing
* Fixed return codes
* Added runtime debug logging

tibold committed:
Asked vcredist not to reboot

tibold committed:
Fixed move leftover

tibold committed:
Changed appveryor to use the pwsh build script.

tibold committed:
Made the inno setup path configurable

tibold committed:
Added appveyor build badge

tibold committed:
Merge pull request #55 from tibold/feature/ci

Got AppVeyor build working again.

tibold committed:
Cleaned up and modularized the build scripts.

GitMensch committed:
added coverity badge to README

GitMensch committed:
qt4 compatibility and cleanup 

* Main.cpp: include qt headers from their include root (including the module directory)
* ThumbnailProvider.cpp [!NDEBUG]: limit includes to the scope they are used for
* ThumbnailProvider.cpp (CThumbnailProvider::GetThumbnail) [QT_VERSION < 0x050200]: retrofixed (`device` somehow lost its dereferencing)

tibold committed:
Merge pull request #56 from GitMensch/coverity

added coverity badge to README

tibold committed:
Merge pull request #57 from GitMensch/qt4

cleanup and qt4 compatibility

tibold committed:
Merge pull request #60 from tibold/feature/build-scripts

Cleaned up and modularized the build scripts.

tibold committed:
Fixed returning the expected error codes

tibold committed:
Fixed correctly zeroing memory.

tibold committed:
Merge pull request #61 from tibold/bug/returncodes

Bug/returncodes

tibold committed:
Added quick logging messages to figure out where does the dll registration fail

tibold committed:
Merge pull request #69 from tibold/bug/dll-register

Fixed locating the embedded Qt plugins

tibold committed:
Updated the installation section in the readme

tibold committed:
Added note about the file explorer options to the troubleshooting.

tibold committed:
Feature/gpg (#70)

* GPG Signing of the installer.

* Fixed signing of the installer

* Fixed build script to automatically overwrite signature on build

tibold committed:
Merge commit 'c7de9df69f905a65d99144347bef162efd30a30a' into develop

tibold committed:
Added samples that are broken.

maphew reviewed (approved):
It all looks good from here. The associated 64bit installer from 1.0.33-nrxlrjev build artifact installed cleanly for me. There's one grammar error in Readme troubleshooting section but not worth making a solo commit for ('neither' should be 'none', there being 3 choices not just 2).

kelderek committed:
Install VC runtime before main setup

Moved the extraction to tmp and execution of the VC runtime from [Files] and [Run] section into [Code].  Running it here allows it to happen before install instead of after, preventing issue #75 where the installer fails to register the DLL.

tibold committed:
Renamed svg_explorer_extension to svg_see

tibold committed:
Added support for compiling with vs2019 and qt 5.15.*
